packages/gestalt/src/HelpButton.js

```diff
@@ -0,0 +1,200 @@
+   */
+  zIndex?: Indexable,
+|};
+
+/**
+ * [HelpButton](https://gestalt.pinterest.systems/web/helpbutton) is an affordance that accompanies an element on the screen. It helps describe or provide assistance on how to use the accompanying element.
+ */
+export default function HelpButton({
+  accessibilityPopoverLabel,
+  idealDirection = 'down',
+  link,
+  onClick,
+  text,
+  zIndex,
+}: Props): Node {
+  const ref = useRef(null);
+  const [isOpen, setIsOpen] = useState(false);
+  const [isHovered, setHovered] = useState(false);
+  const [isFocused, setFocused] = useState(false);
+  const [isActive, setActive] = useState(false);
+  const popoverId = useId();
+  const {
+    accessibilityTooltipMessage,
+    accessibilityLinkLabel: accessibilityDefaultLinkLabel,
+    accessibilityIcon,
+  } = useDefaultLabelContext('HelpButton');
+
+  useEffect(() => {
+    if (isOpen) {
+      ref?.current?.focus();
+    }
+  }, [isOpen]);
+
+  const toggleView = () => {
+    setIsOpen((currVal) => !currVal);
+    setActive((currVal) => !currVal);
+  };
+
+  const onHandleTap = (props) => {
+    toggleView();
+    if (onClick) {
+      onClick(props);
+    }
+  };
+
+  const bgIconColor = isHovered || isActive || isFocused ? 'default' : 'subtle';
+
+  const tooltipZIndex = zIndex ?? new FixedZIndex(1);
+
+  const zIndexLayer = new CompositeZIndex([new FixedZIndex(tooltipZIndex.index())]);
+
+  const textElement = typeof text === 'string' ? <Text align="center">{text}</Text> : text;
+
+  return (
+    <Flex alignItems="center" justifyContent="center" flex="none">
+      <Tooltip
+        text={accessibilityTooltipMessage}
+        idealDirection={idealDirection}
+        zIndex={tooltipZIndex}
+      >
+        <TapArea
+          accessibilityExpanded={isOpen}
+          accessibilityControls={popoverId}
+          accessibilityLabel={accessibilityTooltipMessage}
+          fullWidth={false}
+          onTap={onHandleTap}
+          ref={ref}
+          role="button"
+          rounding="circle"
+          onBlur={() => {
+            setFocused(false);
+            setActive(false);
+          }}
+          onFocus={() => setFocused(true)}
+          onMouseDown={() => setActive(true)}
+          onMouseEnter={() => setHovered(true)}
+          onMouseLeave={() => setHovered(false)}
+          onMouseUp={() => setActive(false)}
+          tabIndex={0}
+        >
+          <Box
+            width={16}
+            height={16}
+            rounding="circle"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            dangerouslySetInlineStyle={{
+              __style: {
+                backgroundColor: `var(--color-text-${bgIconColor})`,
+              },
+            }}
+          >
+            <Icon
```

Comment:
Can we replace Box > Icon with Pog? https://gestalt.pinterest.systems/web/pog#backgroundColorCombinations

Reply:
You dont have to clean up but I "think" you could achieve the same with less code.

Reply:
This is not possible, as you can read on summary:

```
_So, Why we not using Pog?_
Design alignments defined new padding and icon size then we currently used on Pog.
```

Reply:
DONE